dockerfiles/Dockerfile.cross

```diff
@@ -0,0 +1,22 @@
+
+FROM    golang:1.8.1
+
+# allow replacing httpredir or deb mirror
+ARG     APT_MIRROR=deb.debian.org
+RUN     sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
+
+RUN     apt-get update -qq && apt-get install -y -q \
+            libltdl-dev \
+            gcc-mingw-w64 \
+            parallel \
+            ;
```

Comment:
:man_shrugging: , I'm more used to the other way around, I was just wondering if this was adding any side effect I wasn't aware off.